frontend/ui/elements/menu_keyboard_layout.lua

```diff
@@ -1,11 +1,105 @@
+local CheckButton = require("ui/widget/checkbutton")
 local FFIUtil = require("ffi/util")
+local HorizontalGroup = require("ui/widget/horizontalgroup")
+local HorizontalSpan = require("ui/widget/horizontalspan")
 local Language = require("ui/language")
+local UIManager = require("ui/uimanager")
+local VerticalGroup = require("ui/widget/verticalgroup")
 local VirtualKeyboard = require("ui/widget/virtualkeyboard")
+local _ = require("gettext")
 
-local sub_item_table = {}
+local input_dialog, check_button_bold, check_button_border
+
+local sub_item_table = {
+    {
+        text = _("Keyboard layout"),
+        sub_item_table = {},
+    },
+    {
+        text = _("Keyboard labels font size"),
```

Comment:
For the menu title (but also may be for the settings names and variables?), "Keyboard keys font size" ? 
"labels" might be our internal way of calling the label of the keys, dunno if this should show up to the user ?

Reply:
What other alternative is good? `Keyboard letters`?

Reply:
Just `keyboard font size` says all it needs to? No need to add letters/keys/whatever.